New package: About v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: About
- Repository: https://github.com/tecosaur/About.jl
- Created by: @tecosaur
- Version: v1.0.0
- Commit: cf4d5c5f6603a808e207d143c9689f9ee742b47f
- Git reference: HEAD
- Description: Mirror of https://code.tecosaur.net/tec/About.jl

ericphanson commented:
It would be good to add some tests as well

[noblock]

tecosaur commented:
Tests? What are those :stuck_out_tongue: 

(more seriously: most of this package is display code, making testing a bit more annoying)

JuliaHubRegistrator committed:
New package: About v1.0.0

UUID: 69d22d85-9f48-4c46-bbbe-7ad8341ff72a
Repo: https://github.com/tecosaur/About.jl.git
Tree: 3a94c01a1d7678f1c29b7d97216f5046f1ed1768

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

ericphanson commented:
Yeah, really good tests can be pretty tricky there, but you can probably prevent a decent amount of regressions just by printing to a string and checking it contains some word/symbol of interest. That will at least make sure the call doesn’t error and is running the method you expect.

[noblock]
